user/editline/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "editline"
+pkgver = "1.17.1"
+pkgrel = 0
+build_style = "configure"
```

Comment:
this should be gnu_configure and then you can drop the whole thing with cross, configure_args and other nonsense